cosmwasm/Makefile

```diff
@@ -1,78 +1,23 @@
-bridge_SOURCE=wormhole
-token_bridge_SOURCE=token_bridge_terra
-
-SOURCE_FILES=$(shell find . -name "*.rs" -or -name "*.lock" -or -name "*.toml" | grep -v target)
-
 PACKAGES=$(shell find . -name "Cargo.toml" | grep -E 'packages|contracts' | xargs cat | grep "name *=" | cut -d' ' -f3 | sed s/\"//g | sed s/-/_/g)
 WASMS=$(patsubst %, artifacts/%.wasm, $(PACKAGES))
 
 -include ../Makefile.help
 
 .PHONY: artifacts
 ## Build contracts.
-artifacts: artifacts/checksums.txt
```

Comment:
looks like we lost checksums here, not sure if that's a problem, maybe @kcsongor is more familiar

Reply:
this target basically serves as a cache for local development, which will skip building contracts if `checksums.txt` and `wasm` files exist in `artifacts`. It's not a bad thing to have, though it's only for local dev, CI artifacts are cached in the shared container. I'll add this cache back.

Reply:
added this